CHANGELOG.md

```diff
@@ -130,3 +130,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.2.5] - 2024-08-21
```

Comment:
We published this version under `v0.3.0` tag.  Let's update the correct version in the changelog